frontend/app/src/store/balances/exchanges.ts

```diff
@@ -140,7 +140,7 @@ export const useExchangeBalancesStore = defineStore(
             getAssociatedAssetIdentifier
           ),
           asset => hideIgnored && get(isAssetIgnored(asset)),
-          asset => get(prices)[asset]
+          asset => get(prices)[asset]?.value
```

Comment:
The amount of places we had to change this makes me wonder if it would make sense to introduce a function in the store that takes an asset and returns the price instead.

Reply:
Nice idea